layersvt/vulkan_json_layer.hpp

```diff
@@ -879,28 +881,29 @@ class PipelineData {
         std::reverse(tmp.begin(), tmp.end());
         strncpy(m_exeName, tmp.c_str(), sizeof(m_exeName) - 1);
 #endif
-        return m_exeName;
     }
 
-    FILE* openFile() 
+    void getBaseDirectoryPath()
     {
-        char* path = getenv("VK_JSON_FILE_PATH");
-        std::string filename;
-        if (path) {
-            filename = path;
+        m_baseDir = getenv("VK_JSON_FILE_PATH") ? getenv("VK_JSON_FILE_PATH") : "";
+        if (!m_baseDir.empty()) {
 #ifdef _WIN32
-            filename += "\\";
+            m_baseDir += "\\";
 #else
-            filename += "/";
+            m_baseDir += "/";
 #endif
         } else {
-            std::cout << "Error with setting VK_JSON_FILE_PATH. Please set the environment variable.\n";
-            exit(-1);
-        }
-        std::cout << "Dumping generated JSON into:  " << path << "\n";
+	    std::cout << "Error with querying VK_JSON_FILE_PATH. Please set the environment variable.\n";
+	    exit(-1);
+	}
+    }
+
+    FILE* openFile()
+    {
+        std::string filename = m_baseDir + m_exeName;
+        std::cout << "Dumping generated JSON into:  " << filename << "\n";
```

Comment:
nit: This should be std::cout << "Dumping generated JSON into:  " << m_baseDir<< "\n";

Will fix in the next patchset

Reply:
> nit: This should be std::cout << "Dumping generated JSON into: " << m_baseDir<< "\n";
> 
> Will fix in the next patchset

Fixed in https://github.com/KhronosGroup/VulkanTools-ForSC/commit/3ed47da1c8d44c9f005e6c7b674df11f52405313